go/vt/wrangler/reparent.go

```diff
@@ -164,6 +164,18 @@ func (wr *Wrangler) initShardMasterLocked(ctx context.Context, ev *events.Repare
 		return err
 	}
 
+	_, masterTabletMap := topotools.SortedTabletMap(tabletMap)
+
+	// Sanity check that the keyspace / shard info in the topo matches what we passed in to InitShardMaster
+	for _, tablet := range tabletMap {
+		if tablet.GetShard() != shard {
```

Comment:
Shouldn't this check be against `masterElectTabletAlias`  instead of the tabletMap?